cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityOperator.java

```diff
@@ -319,4 +324,31 @@ public Role generateRole(String ownerNamespace, String namespace) {
 
         return role;
     }
+
+    /**
+     * Generates the NetworkPolicies relevant for Entity Operator
+     *
+     * @param operatorNamespace                             Namespace where the Strimzi Cluster Operator runs. Null if not configured.
+     * @param operatorNamespaceLabels                       Labels of the namespace where the Strimzi Cluster Operator runs. Null if not configured.
+     *
+     * @return The network policy.
+     */
+    public NetworkPolicy generateNetworkPolicy(String operatorNamespace, Labels operatorNamespaceLabels) {
```

Comment:
I don't think we care about the `operatorNamespace` and `operatorNamespaceLabels` here as the Cluster operator does not access these metrics / ports.